utils/build-script

```diff
@@ -437,8 +436,6 @@ class BuildScriptInvocation(object):
             impl_args += ["--skip-build-swiftpm"]
         if not args.build_swiftsyntax:
             impl_args += ["--skip-build-swiftsyntax"]
-        if not args.build_skstresstester:
-            impl_args += ["--skip-build-skstresstester"]
         if not args.build_swiftevolve:
             impl_args += ["--skip-build-swiftevolve"]
```

Comment:
I assume you'll handle swift evolve as a follow-up? I would expect it to be almost exactly the same work as handling the stress tester, especially since they're in the same repo.

Reply:
Yeah, should be fairly easy to do, but I wanted to keep this PR small.

Reply:
Works for me